src/storage/concurrency_manager/mod.rs

```diff
@@ -0,0 +1,155 @@
+// Copyright 2020 TiKV Project Authors. Licensed under Apache-2.0.
+
+//! The concurrency manager is responsible for concurrency control of
+//! transactions.
+//!
+//! The concurrency manager contains a key table in memory.
+//! Transactional commands can acquire key mutexes from the concurrency manager
+//! to ensure serializability. Lock information can be also stored in the
+//! manager and reading requests can check if these locks block the read.
```

Comment:
I think the comment is valid but now we don't let it interact with latches. 
Currently we still use latches to ensure serializability of write requests but we are going to replace the latches in the future.

Reply:
I think I find this confusing because the difference between what latches do and what the concurrency manager does is subtle. It would be good for the comment here to reflect what is implemented now, rather than what will be implemented in the future.

Reply:
I update the comment here. It has nothing to do with latch now. I delete the comment talking about using that mutex to ensure serializability.